app/institutions/dashboard/-components/object-list/component.ts

```diff
@@ -277,14 +277,15 @@ export default class InstitutionalObjectList extends Component<InstitutionalObje
     @task
     @waitFor
     async _sendNodeRequest() {
+        const userRecord = await this.store.findRecord('user', this.selectedUserOsfGuid);
         const nodeRequest = this.store.createRecord('node-request', {
             comment: this.messageText.trim(),
             requestType: RequestTypeChoices.InstitutionalRequest,
             requestedPermissions: this.selectedPermissions,
             bccSender: this.bccSender,
             replyTo: this.replyTo,
             institution: this.args.institution,
-            messageRecipient: this.selectedUserOsfGuid,
```

Comment:
This was the most difficult thing, due to mirage I'm still not 100% this is correct.